### PR TITLE
Use #fileID/#filePath instead of #file

### PR DESCRIPTION
**Pull request by carolinacass**
Motivation:

#fileID introduced in Swift 5.3, so no longer need to use #file anywhere                                  

Modifications:

Changed #file to #filePath or #fileID depending on the situation

### CODE REVIEW

File: Sources/Examples/RouteGuide/Client/RouteGuideClient.swift [hunk 1]
```diff
@@ -23,7 +23,7 @@ import RouteGuideModel
 
 /// Loads the features from `route_guide_db.json`, assumed to be in the directory above this file.
 func loadFeatures() throws -> [Routeguide_Feature] {
-  let url = URL(fileURLWithPath: #file)
+  let url = URL(fileURLWithPath: #fileID)
```
Comment (glbrntt) [inline Sources/Examples/RouteGuide/Client/RouteGuideClient.swift:26]: This one should be `#filePath` as we're finding a file relative to this source file.

```suggestion
  let url = URL(fileURLWithPath: #filePath)
```

File: Sources/Examples/RouteGuide/Server/RouteGuideServer.swift [hunk 2]
```diff
@@ -24,7 +24,7 @@ import RouteGuideModel
 
 /// Loads the features from `route_guide_db.json`, assumed to be in the directory above this file.
 func loadFeatures() throws -> [Routeguide_Feature] {
-  let url = URL(fileURLWithPath: #file)
+  let url = URL(fileURLWithPath: #fileID)
```
Comment (glbrntt) [inline Sources/Examples/RouteGuide/Server/RouteGuideServer.swift:27]: Same here: 
```suggestion
  let url = URL(fileURLWithPath: #filePath)
```

**Review by glbrntt — CHANGES_REQUESTED** [review]

**Comment by glbrntt** [comment]
Thanks @carolinacass!

File: Sources/Examples/RouteGuide/Server/RouteGuideServer.swift (hunk 2 above)
Comment (Tharolzakariya) [reply Sources/Examples/RouteGuide/Server/RouteGuideServer.swift]: Withdraw

**Pull request merged**